docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md

```diff
@@ -153,11 +153,11 @@ Player format as follows:
 
 ### Write Configuration Files
 
-The configuration files consist of Spark related information, Nebula Graph related information, and tags mapping and edges mapping blocks. Spark information is configured with the associated parameters running Spark. Nebula Graph information is configured with information such as user name and password to connect Nebula Graph. Tags mapping and edges mapping correspond to the input source mapping of multiple tag/edges respectively, describing the basic information like each tag/edge's data source. It's possible that different tag/edge come from different data sources.
+The configuration files consist of the Spark field, the Nebula Graph field, the tags mapping field, and the edges mapping field. The Spark related parameters are configure in the Spark field. The username and password information for Nebula are configured in the Nebula field. Basic data source information for each tag/edge is described in the tag/edge mapping field. The tag/edge mapping field corresponds to multiple tag/edge inputting sources. Different tag/edge can come from different data sources.
```

Comment:
"The Spark related parameters are configure in the Spark field." ==> "The Spark related parameters are configured in the Spark field."

"for Nebula" ==> "for Nebula Graph"?

"in the Nebula field" ==> "in the Nebula Graph field"

"tag/edge" ==> "tag or edge"